master/master.cfg

```diff
@@ -208,6 +208,7 @@ if PRODUCTION:
         ("AMD64 Clang UBSan", "gps-clang-ubsan", ClangUbsanLinuxBuild, UNSTABLE),
         ("AMD64 Alpine Linux", "ware-alpine", UnixBuild, UNSTABLE),
         ("AMD64 Ubuntu", "einat-ubuntu", UnixBuild, UNSTABLE),
+        ("x86-64 Fedora Rawhide", "cstratak-fedora", UnixBuild, UNSTABLE),
```

Comment:
Maybe rename it to "AMD64 Fedora Rawhide", "AMD64" is used by all 64-bit workers except "x86-64 High Sierra".